lib/model.ts

```diff
@@ -166,6 +218,13 @@ export function initCanvas(
   biggestRgb.listen(value => {
     generateMesh(scene, value)
   })
+  
+  if (!fullControl) {
+    updateSelectors(selectorL, selectorC, selectorH, current.get())
+  }
+  current.listen(value => {
```

Comment:
Why we need to listen current for cull-screen model (we do not need to draw lines on fullscreen)

Reply:
If we are in fullscreen current would not be updating anyway, so it does not influence anything. Shaders are reset to default value, so selectors are not visible in fullscreen

Reply:
But why we need this line in full-screen? Can we move it to `if (!fullControl)`?

Reply:
Sure, I made the change, also automatically fixed some formatting